portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -55,28 +60,30 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
 
-    // Get the input from form text box
+    // Get the input from comment form 
     String text = getParameter(request, "text-input", null);
     String username = getParameter(request, "username", null);
     String email = userService.getCurrentUser().getEmail();
     long timestamp = System.currentTimeMillis();
+    String image = getBlobKey(request);
```

Comment:
I like that you put this functionality of extracting the URL for the image in a separate method.